tabs/src/components/sample/QueryRunner.jsx

```diff
@@ -67,21 +68,37 @@ export function QueryRunner() {
         items: ['Key', 'Value'],
     };
 
-    const callGraph = () => {
-        // stub
+    async function callGraph() {
+        // c7ced617-91ba-4d83-8f69-77c39c6eee68
+        // 19:c213c3137b404a6a83e75a5d5142acaf@thread.tacv2
+        // https://graph.microsoft.com/v1.0/teams/c7ced617-91ba-4d83-8f69-77c39c6eee68/channels/19:c213c3137b404a6a83e75a5d5142acaf@thread.tacv2
+        const queryParameters = query.substring(GRAPH_URL.length + graphVersion.length, query.length);
+        const DEVX_API = "https://graphwebapi.azurewebsites.net/graphproxy/" + graphVersion + queryParameters;
```

Comment:
I know this is still in draft, but its good if we have a constants file that we keep the URL link in